src/vsr/replica.zig

```diff
@@ -3078,8 +3078,10 @@ pub fn ReplicaType(
                 );
             }
 
-            const scrub_next = self.grid_scrubber.read_next();
-            maybe(scrub_next);
+            for (0..constants.grid_scrubber_reads_max + 1) |_| {
+                const scrub_next = self.grid_scrubber.read_next();
+                if (!scrub_next) break;
+            } else unreachable;
```

Comment:
Really loves this pattern with a bounded loop with an unreachable! Which, well, is what our TigerStyle tells us to use, but its only recently that I came to appreciate this fully!